test/functional/examples_tests.js

```diff
@@ -1237,7 +1237,11 @@ describe('Examples', function() {
       const client = configuration.newClient(configuration.writeConcernMax(), { poolSize: 1 });
 
       client.connect(function(err, client) {
+        let session;
         const cleanup = e => {
+          if (session) {
+            session.endSession();
```

Comment:
is this necessary? doesn't `client.close()` imply all sessions will be ended?

Reply:
yes, but for the purposes of our unit tests, it will still be a "leak". I can instead disable the leak checking on this test, since we know it is explicit.